src/main.rs

```diff
@@ -280,15 +280,12 @@ async fn provide_interactive(
     key: Option<PathBuf>,
     keylog: bool,
 ) -> Result<()> {
-    let out_writer = OutWriter::new();
     let keypair = get_keypair(key).await?;
 
     let mut tmp_path = None;
 
     let sources = if let Some(path) = path {
-        out_writer
-            .println(format!("Reading {}", path.display()))
-            .await;
+        println!("Reading {}", path.display());
```

Comment:
Moved "Reading" to `println`. However, leaving this to print to `out_writer` may have been intentional since it is technically "progress".

Easy to switch back, so just say the word.

Reply:
works for me